packages/flame_yarn/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+
+No version has been released yet.
```

Comment:
The changelog shouldn't exist until melos creates it

Reply:
Ok, that's easy to fix :)